staging/src/k8s.io/apiserver/pkg/apis/audit/types.go

```diff
@@ -200,10 +200,27 @@ type GroupResources struct {
 	// The empty string represents the core API group.
 	// +optional
 	Group string
-	// Resources is a list of resources within the API group.
-	// Any empty list implies every resource kind in the API group.
+	// Resources is a list of resources within the API group. Specific subresources
+	// can be matched using a "/" to indicate the subresource. For example "pods" matches
+	// requests to "pods" and "pods/log" but "pods/log" only matches "pods/log".
+	//
+	// To ignore specific subresources, policies should match the subresource before the
+	// non-subresourced kind.
+	//
+	//		policy:
+	//		- level: None
+	// 		  resources: ["pods/log"] # Ignore pods/log.
```

Comment:
Policy format in this example is illegal.
The allowed format which is acceptable to --audit-policy-file option is:
```
  - level: None
    resources:
      - resources: ["pods/log"] # Ignore pods/log.
  - level: Metadata
    resources:
      - resources: ["pods"] # Catch all for "pods" and all other pods subresources.
```
Please take look at the 2nd and 5th line.